gips/datahandler/worker.py

```diff
@@ -140,18 +140,21 @@ def _export(driver, spatial_spec, temporal_spec, products, outdir,
 
     t_extent = TemporalExtent(**temporal_spec)
 
+    errors = []
     for extent in extents:
         inv = DataInventory(DataClass, extent, t_extent, products, **kwargs)
         datadir = os.path.join(tld, extent.site.Value())
         if inv.numfiles > 0:
             inv.mosaic(datadir=datadir, **kwargs)
         else:
+            errors.append((str(extent), str(t_extent)))
             utils.verbose_out(
                 'No data found for {} within temporal extent {}'
-                .format(str(t_extent), str(t_extent)),
-                2,
+                .format(str(extent), str(t_extent)),
+                1,
             )
-    # TODO nothing meaningful to return?
+    if len(errors) > 0:
```

Comment:
OK...so, if null-jobs (jobs which export nothing) are acceptable, then `errors` and the `if` go away.  If null-jobs are not acceptable, then:
```python
if len(errors) == len(extents):
    mark_job_as_failed('nothing to export')
```
Additionally, it could be that the aggregate checks the status of the job before running OR `_export` could `raise ???` or `return false` to indicate to `export_and_aggregate` that it should not attempt `_aggregate`.

What do you guys think?  and oh yeah, conference room ;)